v3.1/how-to/maintain/backup-and-restore/br-best-practice.md

```diff
@@ -0,0 +1,393 @@
+---
+title: BR 快速备份恢复最佳实践
+category: how-to
+---
+
+# BR 快速备份恢复最佳实践
+
+本文介绍了 PingCAP 新推出的分布式快速备份和恢复工具（下文简称 BR） 的原理和使用场景，并且总结了一些场景下的使用经验。
+
+## 面向对象
+
+本文面向所有 BR 的用户，并且最好对 TiDB 和 TiKV 有一定的了解。在阅读本文前推荐通过 BR 使用手册了解基本的备份恢复原理。
+
+## 目标
+
+* 使用网络盘或本地盘进行备份/恢复
+
+* 通过备份/恢复相关指标了解备份/恢复状态
+
+* 了解进行备份/恢复时如何调优性能
+
+* 处理备份时可能发生的异常
+
+## 使用限制
+
+* BR 只支持 TiDB v3.1 及以上版本
+
+* 目前只支持在全新的集群上执行恢复操作
+
+* BR 备份必须串行执行
+
+## 环境准备
+
+Tips：BR 可以直接将命令下发到 TiKV 集群来执行备份和恢复，不需要依赖 tidb-server 组件。
+
+### 部署
+
+* 通过官方推荐的 [tidb-ansible](/v3.1/how-to/deploy/orchestrated/ansible.md) 部署 TiDB 集群
+
+* 通过下载 [tidb-toolkit](/v3.1/reference/tools/download.md#快速备份和恢复br) 获取 br 应用
+
+### 版本
+
```

Comment:
What does the `version` here refer to? Please add a description for it. BTW, why is the lastest `dev` not included?

Reply:
change it to `cluster version`, I can simple copy this doc to `dev` if this pr is ready

Reply:
it's just the version used in this best practices document, br is an independent tool, we can not ask user specify version